components/card/src/Card.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'react-emotion';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { defaultTheme } from '@verdigris/theme';
+import { ThemeProvider } from 'emotion-theming';
+import { withAnalytics } from '@verdigris/analytics';
```

Comment:
A component only needs to use `withAnalytics` if it has event handlers. This one does not seem to have any relevant ones to which this would apply.

Reply:
I was curious about that, good to know